src/schema/entities.yaml

```
@@ -121,6 +121,28 @@ flip:
     denotes the number/index (in the form of a nonnegative integer), not the `FlipAngle`
     value which needs to be stored in the field `FlipAngle` of the separate JSON file.
   format: index
+inv:
+  name: Inversion Time
+  entity: inv
+  description: |
+    If files belonging to an entity-linked file collection are acquired at different
+    inversion times, the `_inv-<index>` key/value pair MUST be used to distinguish
+    individual files.
+    This entity represents the `InversionTime` metadata field. Please note that the `<index>`
+    denotes the number/index (in the form of a nonnegative integer), not the `InversionTime`
+    value which needs to be stored in the field `InversionTime` of the separate JSON file.
+  format: index
+mt:
+  name: Magnetization Transfer
+  entity: mt
+  description: |
+    If files belonging to an entity-linked file collection are acquired at different
+    magnetization transfer (MT) states, the `_mt-<label>` key/value pair MUST be used to 
+    distinguish individual files.
+    This entity represents the `MTState` metadata field. Allowed label values for this 
```

removing trailing spaces for the linter

```suggestion
    magnetization transfer (MT) states, the `_mt-<label>` key/value pair MUST be used to
    distinguish individual files.
    This entity represents the `MTState` metadata field. Allowed label values for this
```